frontend/src/react/Correction/CopiesOfExamDisplayer.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class CopiesOfExamDisplayer extends Component {
+  render () {
+    const copies = this.props.copies.length > 0 && this.props.copies.map((copy) => {
+      const examId = this.props.examId;
+      return (
+      <Link to={`/correction/${examId}/copy/${copy._id}`} className="content pointer" key={copy._id}>
+        <div className="card">
+          <header className="card-header">
+            <div className="card-header-title has-no-margin-bottom">
+              <div className="title is-5" style={{color: '#FFF'}}>Author: </div> {copy._id} 
```

Comment:
If we had time, we should use "has-text-white", which is a bulma class to have white text

Reply:
yup, I'm just lazy